cylc/flow/task_events_mgr.py

```diff
@@ -1465,7 +1459,6 @@ def _process_message_submit_failed(
             "time_submit_exit": event_time,
             "submit_status": 1,
         })
-        itask.summary['submit_method_id'] = None
```

Comment:
Don't know why the job ID was being wiped on submit-failure

Reply:
Is this the job ID or the `job runner`?